docs/README.md

```diff
@@ -0,0 +1,23 @@
+# README
+The [OSV-Scanner docs](https://google.github.io/osv-scanner) are hosted on a [GitHub page](https://pages.github.com/). 
+
+Changes in documentation, new features, etc. are added to the [working docs](working_docs/). Docs in the working folder do not represent the current release of OSV-Scanner.  
+
+## Running docs locally
+To run docs locally, you will need [Jekyll](https://jekyllrb.com/docs/installation/) on your machine. 
+
+Here are other [pre-requisites] and instructions for running the [docs locally].
+
+[pre-requisites]: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll#prerequisites
+[docs locally]: https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll#building-your-site-locally
+
+## Contributing to the docs
+Documentation contributions are welcome!
+
+Contributors should make changes to the documentation in [working docs](working_docs/).
+
+See [CONTRIBUTING.md](../CONTRIBUTING.md) for contribution guidelines. 
```

Comment:
Can you update CONTRIBUTING.md with a line or two about contributing to docs?

Also full link to github repo file is probably required here.

Reply:
Done! This link works because it is in the readme, while the others did not because they were on the github page.